internal/stage_common.go

```diff
@@ -18,3 +23,39 @@ func randSeq(n int) string {
 	}
 	return string(b)
 }
+
+func sendRequest(client *http.Client, req *http.Request, logger *testerutils.Logger) (*http.Response, error) {
+	reqDump, err := httputil.DumpRequestOut(req, true)
+	if err != nil {
+		return nil, err
+	}
+	logger.Debugf("Sending Request:\n%s", string(reqDump))
```

Comment:
@sherubthakur I'm not sure how verbose this is, but I think we should have "INFO" logs that show a basic line at least (like `Sending request: GET /abcd`). Doesn't have to include headers and what not - this just give the user a rough idea of what the program is doing. 

Debug logs can be verbose (more the better really, wouldn't hold back).